### ISSUE TITLE
RAID information isn't displayed on Ubuntu 14.04

### ISSUE THREAD

**Issue by rakesh-sankar** [nicolargo/glances#521]
The RAID 1 disks information on Ubuntu 12.02 displayed properly, after the OS upgrade to 14.04, I do not see the RAID disks information on the glances. The glances version was 2.2.1. Please help.

**Comment by nicolargo** [comment]
Hi @rakesh-sankar 

The RAID plugin is only available in the version 2.3 (or higher) not in the 2.2.1.

How do you install Glances on your system ?
If you use the [autoinstall script](https://github.com/nicolargo/glancesautoinstall), please run it again to upgrade Glances and all its dependencies.

**Comment by rakesh-sankar** [comment]
Ah, my bad, it is glances 2.3 (that was a typo).

I used `sudo apt-get to install glances`, the autoinstall script fixed it. Looks like one of the dependencies were not installed/missing (pysensors(?) - see below) from the apt-get installation.

```bash
Requirement already up-to-date: psutil in /usr/local/lib/python2.7/dist-packages
Requirement already up-to-date: bottle in /usr/local/lib/python2.7/dist-packages
Requirement already up-to-date: batinfo in /usr/local/lib/python2.7/dist-packages
Requirement already up-to-date: zeroconf in /usr/local/lib/python2.7/dist-packages
Requirement already up-to-date: netifaces in /usr/local/lib/python2.7/dist-packages
Requirement already up-to-date: pymdstat in /usr/local/lib/python2.7/dist-packages
Requirement already up-to-date: influxdb in /usr/local/lib/python2.7/dist-packages
Requirement already up-to-date: statsd in /usr/local/lib/python2.7/dist-packages
Requirement already up-to-date: pystache in /usr/local/lib/python2.7/dist-packages
Requirement already up-to-date: docker-py in /usr/local/lib/python2.7/dist-packages
Requirement already up-to-date: pysnmp in /usr/local/lib/python2.7/dist-packages
Downloading/unpacking requests>=1.0.3 from https://pypi.python.org/packages/py2.py3/r/requests/requests-2.5.3-py2.py3-none-any.whl#md5=233249f4627ac5481c948e494d2a090e (from influxdb)
  Downloading requests-2.5.3-py2.py3-none-any.whl (468kB): 468kB downloaded
Requirement already up-to-date: six==1.9.0 in /usr/local/lib/python2.7/dist-packages (from influxdb)
Requirement already up-to-date: websocket-client>=0.11.0 in /usr/local/lib/python2.7/dist-packages (from docker-py)
Requirement already up-to-date: pyasn1>=0.1.2 in /usr/local/lib/python2.7/dist-packages (from pysnmp)
Requirement already up-to-date: pycrypto>=2.4.1 in /usr/local/lib/python2.7/dist-packages (from pysnmp)
Requirement already up-to-date: backports.ssl-match-hostname in /usr/local/lib/python2.7/dist-packages (from websocket-client>=0.11.0->docker-py)
Installing collected packages: PySensors, requests
  Found existing installation: PySensors 0.0.2
    Uninstalling PySensors:
      Successfully uninstalled PySensors
  Running setup.py install for PySensors

    warning: no files found matching '*.txt' under directory 'docs'
  Found existing installation: requests 2.2.1
    Uninstalling requests:
      Successfully uninstalled requests
Successfully installed PySensors requests
```
